runtime/parachains/src/lib.rs

```diff
@@ -49,11 +50,11 @@ pub use paras::ParaLifecycle;
 pub fn schedule_para_initialize<T: paras::Config>(
 	id: primitives::v1::Id,
 	genesis: paras::ParaGenesisArgs,
-) {
-	<paras::Module<T>>::schedule_para_initialize(id, genesis);
+) -> Result<(), ()> {
+	<paras::Module<T>>::schedule_para_initialize(id, genesis).map_err(|_| ())
```

Comment:
An API nit. When you want to do swapping I assume you'd call `onboard()` on one and `offboard()` on the other. The first might succeed and the second might fail. You could either have some kind of atomic updating API where you update multiple paras at once or you just expose the `can_*` functions here. Something for a follow-up?

Reply:
ack, will follow up in another PR and with more consideration specifically with the swap integration in registrar